_faq.Rmd

```diff
@@ -344,12 +359,21 @@ phoible %>%
 
 # Why are multiple inventories sometimes linked to the same document?
 
-Occasionally, a single document may provide information about multiple phonological inventories. For example, a dissertation that describes and compares three related dialects or languages spoken in a particular region. In that case, three phonological inventories in PHOIBLE corresponding to three doculects might all be linked to the same document, but each inventory is still linked to only one document (it just happens to be *the same document* for those three inventories). One example is @Terrill1998, a grammar of [Biri](https://en.wikipedia.org/wiki/Biri_language), which contains a chapter describing its dialects, [many of which appear in PHOIBLE](https://phoible.org/languages/biri1256).
+Occasionally, a single document may provide information about multiple
+phonological inventories. For example, a dissertation that describes and
+compares three related dialects or languages spoken in a particular region. In
+that case, three phonological inventories in PHOIBLE corresponding to three
+doculects might all be linked to the same document, but each inventory is still
+linked to only one document (it just happens to be *the same document* for
+those three inventories). One example is @Terrill1998, a grammar of
+[Biri](https://en.wikipedia.org/wiki/Biri_language), which contains a chapter
+describing its dialects, [many of which appear in
+PHOIBLE](https://phoible.org/languages/biri1256).
 
 
 # What are the different "sources" in PHOIBLE?
 
-PHOIBLE contains inventories from various different
+PHOIBLE contains inventories from various
 [contributors](https://phoible.org/contributors). These contributions are
 grouped into so-called "sources", denoted by abbreviations. Here they
 are in the chronological order that they were added to PHOIBLE:
```

Comment:
This list is out of order, if we say that it's in chronological order. It would be:

* SPA: The Stanford Phonology Archive (Crothers, Lorentz, Sherman, & Vihman, 1979)
* UPSID: The UCLA Phonological Segment Inventory Database (Maddieson, 1984; Maddieson & Precoda, 1990)
* AA: Alphabets of Africa (Chanard, 2006; Hartell, 1993)
* PH: Data drawn from journal articles, theses, and published grammars, added by members of the Linguistic Phonetics Laboratory at the University of Washington (Moran, 2012)
* GM: Data from African and Southeast Asian languages
* RA: Common Linguistic Features in Indian Languages (Ramaswami, 1999)
* SAPHON: South American Phonological Inventory Database (Michael, Stark, & Chang, 2012)
* UZ: Data drawn from journal articles, theses, and published grammars, added by the phoible developers while at the Department of Comparative Linguistics at the University of Zurich
* EA: The database of Eurasian phonological inventories (beta version) (Nikolaev, Nikulin, & Kukhto, 2015)
* ER: Australian phonemic inventories (Round, 2019)

Also, below maybe we should add some text before the code snippet, e.g.

Here is code to calculate the number of inventories per source.